### PR TITLE
feat: activity list calculated review status + filtering + sorting

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3311368&stafilter=NotComplete&fileview=grid

Signed-off-by: romanetar <roman_ag@hotmail.com>

### CODE REVIEW

File: app/Models/Foundation/Summit/Summit.php [hunk 1]
```diff
@@ -2376,7 +2376,7 @@ public function getSlug($suffix = '-summit')
     public function generateRegistrationSlugPrefix(): void
     {
         if (empty($this->registration_slug_prefix)) {
-            $this->registration_slug_prefix = $this->slug;
+            $this->registration_slug_prefix = $this->getSlug();
```
Comment (smarcet) [inline app/Models/Foundation/Summit/Summit.php:2379]: why this change is neeeded ? slug is entered ny user not generated please rollback it back

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review comments

File: app/Models/Foundation/Summit/Summit.php (hunk 1 above)
Comment (romanetar) [reply app/Models/Foundation/Summit/Summit.php]: rolled back

File: app/Models/Foundation/Summit/Events/Presentations/Presentation.php [hunk 2]
```diff
@@ -2402,4 +2418,77 @@ public function getSnapshot():array{
...
+        // check submission period
+        $submission_open = $now >= $submission_begin_date && $now <= $submission_end_date;
+
+        // selection ( track chairs )
+        // check selection period
+        $selection_open = $now >= $selection_begin_date && $now <= $selection_end_date;
+
+        $presentation_status = $this->getStatus();
+
+        if (!in_array($presentation_status, [self::ReviewStatusReceived, self::ReviewStatusAccepted])) {
```
Comment (smarcet) [inline app/Models/Foundation/Summit/Events/Presentations/Presentation.php:2460]: @romanetar please lets not change the logic , lets be sure to follow the same logic as https://github.com/fntechgit/call-for-presentations/blob/b1952301be6962feb89913448bfdd4543c527823/src/model/presentation.js#L84

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 3]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
...
+                    Presentation::ReviewStatusNoSubmitted => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusNoSubmitted,
+                        "selp IS NULL OR (p.status <> 'Received' AND p.status <> 'Accepted')"
+                    ),
+                    Presentation::ReviewStatusReceived => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusReceived,
+                        "p.status = 'Received' AND e.published = 0 AND
+                            selp IS NOT NULL AND ( 
+                                (selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND 
+                                selp.submission_begin_date <= '$now_utc' AND
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:540]: dont use now utc like this use a DATABASE TIMESTAMP

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 4]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
...
+                        "p.status = 'Received' AND e.published = 0 AND
+                            selp IS NOT NULL AND ( 
+                                (selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND 
+                                selp.submission_begin_date <= '$now_utc' AND 
+                                selp.submission_end_date >= '$now_utc' AND
+                                selp.selection_begin_date <= '$now_utc' AND 
+                                selp.selection_end_date >= '$now_utc'
+                            )"
+                    ),
+                    Presentation::ReviewStatusInReview => new DoctrineCaseFilterMapping(
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:546]: checked ( OK )

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 5]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
...
+                                (selp.submission_begin_date > '$now_utc' OR selp.submission_end_date < '$now_utc') 
+                                AND 
+                                (selp.selection_begin_date <= '$now_utc' AND selp.selection_end_date >= '$now_utc')
+                            )
+                         )"
+                    ),
+                    Presentation::ReviewStatusPublished => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusPublished,
+                        "(p.status = 'Received' OR p.status = 'Accepted') AND e.published = 1 AND
+                            selp IS NOT null AND (
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:561]: i don think this past make sense
should be only
"(p.status = 'Received' OR p.status = 'Accepted') AND e.published = 1

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 6]
```diff
@@ -616,12 +697,61 @@ protected function getOrderMappings()
...
+		(selp.submission_lock_down_presentation_status_date IS NOT NULL AND selp.submission_lock_down_presentation_status_date > '$now_utc') OR 
+		(
+			(selp.submission_begin_date > '$now_utc' OR selp.submission_end_date < '$now_utc') 
+			AND 
+			(selp.selection_begin_date <= '$now_utc' AND selp.selection_end_date >= '$now_utc')
+		)
+	) THEN 'InReview'
+	WHEN (p.status = 'Received' OR p.status = 'Accepted') AND e.published = 1 AND
+		selp IS NOT null AND ( 
+		(selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:721]: please review https://github.com/OpenStackweb/summit-api/pull/263/files#r1610591929

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review comments

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 7]
```diff
@@ -531,6 +610,8 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
      */
     protected function getOrderMappings()
     {
+        $now_utc = date_format(new \DateTime('now', new \DateTimeZone('UTC')), "Y/m/d H:i:s");
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:613]: check
https://github.com/OpenStackweb/summit-api/pull/263/files#r1610577282

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 8]
```diff
@@ -616,12 +697,61 @@ protected function getOrderMappings()
...
+            SELECT ___sp332.id
+            FROM models\summit\SummitSelectedPresentation ___sp332
+            JOIN ___sp332.presentation ___p332
+            JOIN ___p332.category ___pc332                
+            JOIN ___sp332.list ___spl332 WITH ___spl332.list_type = 'Group' AND ___spl332.list_class = 'Session'
+            WHERE ___p332.id = e.id
+            AND ___sp332.collection = 'selected'
+            AND ___sp332.order > ___pc332.session_count                  
+    ) THEN 'Rejected'
+    ELSE 'Received'
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:752]: else should https://github.com/fntechgit/call-for-presentations/blob/b1952301be6962feb89913448bfdd4543c527823/src/model/presentation.js#L137C43-L137C56
please review the logic

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 9]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
                     WHERE p4:i.id = p.id AND mut2:i.id :operator :value
                 )'
             ),
+            'review_status' => new DoctrineSwitchFilterMapping
+            (
+                [
+                    Presentation::ReviewStatusNoSubmitted => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusNoSubmitted,
+                        "selp IS NULL OR (p.status <> 'Received' AND p.status <> 'Accepted')"
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:533]: please do https://github.com/OpenStackweb/summit-api/blob/2d46fc910d303115fc680f5dc9763acbf68728bf/app/Repositories/Summit/DoctrineSummitEventRepository.php#L307
p.status is null

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 10]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
...
+                            (selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND 
+                            (
+                                (selp.submission_begin_date <= '$now_utc' AND selp.submission_end_date >= '$now_utc') 
+                                OR selp.selection_begin_date > '$now_utc' OR selp.selection_end_date < '$now_utc'
+                            )
+                        )"
+                    ),
+                    Presentation::ReviewStatusAccepted => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusAccepted,
+                        sprintf('(p.status = \'Received\' OR p.status = \'Accepted\') AND e.published = 0 AND
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:571]: this e.published  = 0 is not correct ( review accepted status ) https://github.com/OpenStackweb/summit-api/blob/2d46fc910d303115fc680f5dc9763acbf68728bf/app/Repositories/Summit/DoctrineSummitEventRepository.php#L378

File: app/Models/Foundation/Summit/Events/Presentations/Presentation.php [hunk 11]
```diff
@@ -2402,4 +2418,77 @@ public function getSnapshot():array{
...
         return array_merge( $snapshot, parent::getSnapshot());
     }
+
+    /**
+     * @return string
+     * @throws \Exception
+     */
+    public function getReviewStatus(): string
+    {
+        if ($this->selection_plan == null) return self::ReviewStatusNoSubmitted;
```
Comment (smarcet) [inline app/Models/Foundation/Summit/Events/Presentations/Presentation.php:2428]: @romanetar please lets not change the logic , lets be sure to follow the same logic as https://github.com/fntechgit/call-for-presentations/blob/b1952301be6962feb89913448bfdd4543c527823/src/model/presentation.js#L84

File: app/Models/Foundation/Summit/Events/Presentations/Presentation.php (hunk 2 above)
Comment (smarcet) [inline app/Models/Foundation/Summit/Events/Presentations/Presentation.php:2460]: this is to much implicit behavior please do check the original implementation
we need to write code that communicate so it could be maintanable

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 12]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
...
+                [
+                    Presentation::ReviewStatusNoSubmitted => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusNoSubmitted,
+                        "selp IS NULL OR (p.status <> 'Received' AND p.status <> 'Accepted')"
+                    ),
+                    Presentation::ReviewStatusReceived => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusReceived,
+                        "p.status = 'Received' AND e.published = 0 AND
+                            selp IS NOT NULL AND ( 
+                                (selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:539]: NOT ( lockDownPeriod OR (submissionClosed AND selectionOpen) ) by DE Morgan law is 
NOT lockDownPeriod AND NOT ( submissionClosed AND selectionOpen) =
NOT lockDownPeriod AND ( NOT  submissionClosed OR NOT selectionOpen) =
NOT lockDownPeriod AND ( submissionOpen OR selectionClosed )
please review expression

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 13]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
...
+                                (selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND 
+                                selp.submission_begin_date <= '$now_utc' AND 
+                                selp.submission_end_date >= '$now_utc' AND
+                                selp.selection_begin_date <= '$now_utc' AND 
+                                selp.selection_end_date >= '$now_utc'
+                            )"
+                    ),
+                    Presentation::ReviewStatusInReview => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusInReview,
+                        "(p.status = 'Received' OR p.status = 'Accepted') AND
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:548]: this is OK

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 14]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
...
+                                AND 
+                                (selp.selection_begin_date <= '$now_utc' AND selp.selection_end_date >= '$now_utc')
+                            )
+                         )"
+                    ),
+                    Presentation::ReviewStatusPublished => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusPublished,
+                        "(p.status = 'Received' OR p.status = 'Accepted') AND e.published = 1 AND
+                            selp IS NOT null AND ( 
+                            (selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:562]: this should be the same as https://github.com/OpenStackweb/summit-api/pull/263/files#r1610819526 and here seems to be following the demorgan law

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 15]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
...
+                            AND ___sp32.collection = \'%2$s\'
+                        )',
+                        $now_utc,
+                        SummitSelectedPresentation::CollectionSelected,
+                        SummitSelectedPresentationList::Group,
+                        SummitSelectedPresentationList::Session)
+                    ),
+                    Presentation::ReviewStatusRejected => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusRejected,
+                        sprintf('(p.status = \'Received\' OR p.status = \'Accepted\') AND e.published = 0 AND
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:588]: this e.published = 0 is not correct check https://github.com/OpenStackweb/summit-api/pull/263/files#r1610638021

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 16]
```diff
@@ -616,12 +697,61 @@ protected function getOrderMappings()
...
+            'actions' => 'HIDDEN_COMPLETED_ACTIONS_COUNT',
             /*
             'event_type_capacity' => <<<SQL
 SQL,
             'speakers' => <<<SQL
 SQL,*/
+            'review_status' => <<<SQL
+    CASE
+    WHEN p is null THEN ''
+	WHEN selp IS NULL OR (p.status <> 'Received' AND p.status <> 'Accepted') THEN 'NoSubmitted'
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:709]: here just check for p.status is null

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 6 above)
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:721]: check https://github.com/OpenStackweb/summit-api/pull/263/files#r1610819526

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 8 above)
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:752]: do not set received as else
please do check the proper conditions as its doing on the filtering logic
* ( p.status = 'Received' OR p.status = 'Accepted') AND
* NOT lockDownPeriod AND ( submissionOpen OR selectionClosed ) AND
* e.published = 0 AND selectionClosed

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 17]
```diff
@@ -616,12 +697,61 @@ protected function getOrderMappings()
...
+            FROM models\summit\SummitSelectedPresentation ___sp331 
+            JOIN ___sp331.presentation ___p331 
+            JOIN ___p331.category ___pc331                
+            JOIN ___sp331.list ___spl331 WITH ___spl331.list_type = 'Group' AND ___spl331.list_class = 'Session' 
+            WHERE ___p331.id = e.id 
+            AND ___sp331.collection = 'selected'
+            AND ___sp331.order > ___pc331.session_count                  
+    ) THEN 'Accepted'
+    WHEN (p.status = 'Received' OR p.status = 'Accepted') AND e.published = 0 AND 
+        (selp.selection_begin_date > '$now_utc' OR selp.selection_end_date < '$now_utc')
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:741]: u need to check here the proper conditions for
NOT lockDownPeriod AND ( submissionOpen OR selectionClosed )

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 18]
```diff
@@ -616,12 +697,61 @@ protected function getOrderMappings()
...
+		selp IS NOT null AND ( 
+		(selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND 
+		(
+			(selp.submission_begin_date <= '$now_utc' AND selp.submission_end_date >= '$now_utc') 
+			OR 
+			(selp.selection_begin_date > '$now_utc' OR selp.selection_end_date < '$now_utc')
+		)
+	) THEN 'Published'
+    WHEN (p.status = 'Received' OR p.status = 'Accepted') AND e.published = 0 AND 
+        (selp.selection_begin_date > '$now_utc' OR selp.selection_end_date < '$now_utc')
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:729]: u need to check here the proper conditions for
NOT lockDownPeriod AND ( submissionOpen OR selectionClosed )

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 19]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
...
+                            (
+                                (selp.submission_begin_date <= '$now_utc' AND selp.submission_end_date >= '$now_utc') 
+                                OR selp.selection_begin_date > '$now_utc' OR selp.selection_end_date < '$now_utc'
+                            )
+                        )"
+                    ),
+                    Presentation::ReviewStatusAccepted => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusAccepted,
+                        sprintf('(p.status = \'Received\' OR p.status = \'Accepted\') AND e.published = 0 AND 
+                        (selp.selection_begin_date > \'%1$s\' OR selp.selection_end_date < \'%1$s\') AND
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:572]: u need to check here the proper conditions for
NOT lockDownPeriod AND ( submissionOpen OR selectionClosed )

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 20]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
...
+                        )',
+                        $now_utc,
+                        SummitSelectedPresentation::CollectionSelected,
+                        SummitSelectedPresentationList::Group,
+                        SummitSelectedPresentationList::Session)
+                    ),
+                    Presentation::ReviewStatusRejected => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusRejected,
+                        sprintf('(p.status = \'Received\' OR p.status = \'Accepted\') AND e.published = 0 AND
+                        (selp.selection_begin_date > \'%1$s\' OR selp.selection_end_date < \'%1$s\') AND
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:589]: u need to check here the proper conditions for
NOT lockDownPeriod AND ( submissionOpen OR selectionClosed )

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 21]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
...
+                            (
+                                (selp.submission_begin_date > '$now_utc' OR selp.submission_end_date < '$now_utc') 
+                                AND 
+                                (selp.selection_begin_date <= '$now_utc' AND selp.selection_end_date >= '$now_utc')
+                            )
+                         )"
+                    ),
+                    Presentation::ReviewStatusPublished => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusPublished,
+                        "(p.status = 'Received' OR p.status = 'Accepted') AND e.published = 1 AND
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:560]: lets move it to the 3rd place
so if the 2nd place of the case is in review
then we now that we dont need to check the negation of 
NOT ( lockDownPeriod OR (submissionClosed AND selectionOpen) 
bc was tested on 2nd place
and pass ( check how the case function works 
see https://www.w3schools.com/sql/func_mysql_case.asp 
The CASE statement goes through conditions and return a value when the first condition is met (like an IF-THEN-ELSE statement). So, once a condition is true, it will stop reading and return the result.

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 6 above)
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSummitEventRepository.php:721]: this part is not needed 
check https://github.com/OpenStackweb/summit-api/pull/263/files#r1610829832

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 3 above)
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSummitEventRepository.php:540]: done

File: app/Repositories/Summit/DoctrineSummitEventRepository.php [hunk 22]
```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
...
+            'review_status' => new DoctrineSwitchFilterMapping
+            (
+                [
+                    Presentation::ReviewStatusNoSubmitted => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusNoSubmitted,
+                        "selp IS NULL OR (p.status <> 'Received' AND p.status <> 'Accepted')"
+                    ),
+                    Presentation::ReviewStatusReceived => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusReceived,
+                        "p.status = 'Received' AND e.published = 0 AND
```
[the remark this answers is not in the document]
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSummitEventRepository.php:537]: done

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 7 above)
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSummitEventRepository.php:613]: done

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 9 above)
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSummitEventRepository.php:533]: done

File: app/Models/Foundation/Summit/Events/Presentations/Presentation.php (hunk 2 above)
Comment (romanetar) [reply app/Models/Foundation/Summit/Events/Presentations/Presentation.php:2460]: done

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 12 above)
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSummitEventRepository.php:539]: done

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 13 above)
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSummitEventRepository.php:548]: done

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 14 above)
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSummitEventRepository.php:562]: done

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 16 above)
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSummitEventRepository.php:709]: done

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 21 above)
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSummitEventRepository.php:560]: done

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 8 above)
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSummitEventRepository.php:752]: done

File: app/Repositories/Summit/DoctrineSummitEventRepository.php (hunk 6 above)
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSummitEventRepository.php]: done

**Review by smarcet — APPROVED** [review]
LGTM
